src/Domain/Package.php

```diff
@@ -0,0 +1,139 @@
+        public int $active,
+        public int $global,
+        public ?string $urlID,
+        public string $notifyMarketing,
+        public string $productType,
+        public string $type,
+        public string $contentType,
+        public int $offerRebillMonths,
+        public int $offerRebillActive,
+        public int $trialDays,
+        public ?string $imageURL,
+        public bool $requirePurchasedDomain,
+        public bool $allowMultiplePurchase,
+        public bool $showInStore,
+        public bool $showInTemplatePicker,
+        public ?string $bannerHTML,
+        public string $affiliateLink,
+        public ?string $flowName,
+        public ?array $metadata,
+        public int $brandRef,
+        public string $brandName,
+        public int $defaultCurrencyRef,
+        public string $currencyCode,
+        public string $currencyName,
+        public string $currencyTitle,
+        public ?int $defaultCampaignRef,
+        public array $capabilities,
+        public ?int $templateGroupRef,
+        public ?array $templateGroup,
+        public ?int $displayOrder,
+        public ?DomainProduct $domainProduct,
+        public array $prices,
+        public array $plugins,
+    ) {
+    }
+
+    /**
+     * @inheritDoc
+     */
+    public function toArray(): array
+    {
+        return [
+            'ref' => $this->ref,
+            'name' => $this->name,
+            'active' => $this->active,
+            'global' => $this->global,
+            'urlID' => $this->urlID,
+            'notifyMarketing' => $this->notifyMarketing,
+            'productType' => $this->productType,
+            'type' => $this->type,
+            'contentType' => $this->contentType,
+            'offerRebillMonths' => $this->offerRebillMonths,
+            'offerRebillActive' => $this->offerRebillActive,
+            'trialDays' => $this->trialDays,
+            'requirePurchasedDomain' => $this->requirePurchasedDomain,
+            'allowMultiplePurchase' => $this->allowMultiplePurchase,
+            'showInStore' => $this->showInStore,
+            'showInTemplatePicker' => $this->showInTemplatePicker,
+            'bannerHTML' => $this->bannerHTML,
+            'affiliateLink' => $this->affiliateLink,
+            'flowName' => $this->flowName,
+            'metadata' => $this->metadata,
+            'brandRef' => $this->brandRef,
+            'brandName' => $this->brandName,
+            'defaultCurrencyRef' => $this->defaultCurrencyRef,
+            'currencyCode' => $this->currencyCode,
+            'currencyName' => $this->currencyName,
+            'currencyTitle' => $this->currencyTitle,
+            'defaultCampaignRef' => $this->defaultCampaignRef,
+            'capabilities' => $this->capabilities,
+            'templateGroupRef' => $this->templateGroupRef,
+            'templateGroup' => $this->templateGroup,
+            'displayOrder' => $this->displayOrder,
+            'domainProduct' => $this->domainProduct,
```

Comment:
See my comment on the to array call on AccountPackage class. Same thing here, if you would agree that you also would expect it do be done recursively, then I would suggest the same change here:
```suggestion
            'domainProduct' => $this->domainProduct->toArray(),
```

And if you don't think this should be done recursively, then it is good as it currently is.